torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -42,7 +42,10 @@ RegisterOperators reg({
             autograd::profiler::RecordFunction record("FusionGroup");
             std::vector<at::Tensor> toutputs;
             // TODO: have fusion_fn work off of a stack as well
-            fusion_fn->launch(last(stack, num_inputs), toutputs);
+            auto tinputs = fmap(last(stack, num_inputs), [](const IValue& v) {
+              return v.toTensor();
+            });
```

Comment:
It's a bit annoying the we have all those new allocations 😕

Reply:
Agreed, and I outlined some options above.